lib/matplotlib/widgets.py

```diff
@@ -809,6 +809,24 @@ def on_changed(self, func):
         """
         return self._observers.connect('changed', lambda val: func(val))
 
+    def update_range(self, vmin=None, vmax=None):
+        """Update the range of the slider"""
+        if not vmin and not vmax:
+            raise ValueError(
+                (f"Argument vmin ({type(vmin)}) has no value"
+                    f"Argument vmax ({type(vmax)}) has no value"))
```

Comment:
```suggestion
            return
```
Not changing any value can be a reasonable use case when the parameters are not in the callers control. We can gracefully do nothing, otherwise the caller would have to manually do this check.

Reply:
Done

Reply:
Please remove raising the ValueError. We don't need to raise here, and the return after the raise is never reached.